nested.just

```diff
@@ -48,6 +53,11 @@ simulate whichSafe hdPath='0':
 sign whichSafe hdPath='0':
   #!/usr/bin/env bash
   bundlePath=${taskPath}/${bundleName}.json
+  script=NestedSignFromJson
+  if [ -f "${taskPath}/NestedSignFromJson.s.sol" ]; then
```

Comment:
I don't think this line is working correctly.

 Can you add an echo here so we are sure its hitting the script?

```bash
echo "Running script with assertions"
```

Reply:
I don't think `taskPath` is correct, i think you want `$PWD`